install_handler.js

```diff
@@ -2,26 +2,27 @@
 // Helpers for installing and removing versions
 //
 // TODO: the installing functions are still in renderer.js and should be moved here
-const fs = require('fs');
+const fs = require('fs-extra');
```

Comment:
The use of `fs-extra` here is unnecessary, as all methods used from it are available in `fs`.

Reply:
Is this still the case? Even if fs-extra is needed the variable it is assigned to should be `fsExtra` to be clear that it isn't `fs`